src/utils/toasts.jsx

```diff
@@ -1,26 +1,19 @@
-import Toast from "components/Toast"
-import { toast } from "react-toastify"
-
-import elementStyles from "styles/isomer-cms/Elements.module.scss"
+import { useToast } from "@opengovsg/design-system-react"
 
 import { DEFAULT_ERROR_TOAST_MSG } from "utils"
 
-export function errorToast(message) {
-  return toast(
-    <Toast
-      notificationType="error"
-      text={message || DEFAULT_ERROR_TOAST_MSG}
-    />,
-    {
-      className: `${elementStyles.toastError} ${elementStyles.toastLong}`,
-      toastId: "error",
-    }
-  )
-}
+export const useErrorToast = () =>
+  useToast({
+    description: DEFAULT_ERROR_TOAST_MSG,
+    status: "danger",
+    duration: 5000,
+    isClosable: true,
+  })
 
-export function successToast(message) {
-  toast(<Toast notificationType="success" text={message || `Success!`} />, {
-    className: `${elementStyles.toastSuccess}`,
-    toastId: "success",
+export const useSuccessToast = () =>
+  useToast({
+    description: "Success!",
+    status: "success",
+    duration: 5000,
+    isClosable: true,
```

Comment:
could we have a `DEFAULT_TOAST_PROPS` and then use it for these 2? also wdyt about adding a `useWarningToast` as the toast in design system has 3 variants (but it'll be unused so i'm fine w/ not adding also)